File: app/src/main/java/com/example/whattowatch/model/mappers/DetailMapper.java
package com.example.whattowatch.model.mappers;

import com.example.whattowatch.model.mymodel.MyDetailModel;
import com.example.whattowatch.model.response.detail.DetailMovie;

public class DetailMapper {

    public static MyDetailModel convertToMyDetailModel(DetailMovie detail){
        return new MyDetailModel(
                detail.getId(),
                detail.getTitle(),
                detail.getPosterPath(),
                detail.getBackdropPath(),
                detail.getRuntime(),
                detail.getVoteAverage().floatValue(),
                detail.getOverview(),
                detail.getReleaseDate());
    }
}

File: app/src/main/java/com/example/whattowatch/model/mappers/DetailMapper.kt
package com.example.whattowatch.model.mappers

import com.example.whattowatch.model.mymodel.MyDetailModel
import com.example.whattowatch.model.response.detail.DetailMovie

object DetailMapper {
    fun convertToMyDetailModel(detail: DetailMovie): MyDetailModel {
        return MyDetailModel(
                detail.id,
                detail.title,
                detail.posterPath,
                detail.backdropPath,
                detail.runtime,
                detail.voteAverage?.toFloat(),
                detail.overview,
                detail.releaseDate)
    }
}